### ISSUE TITLE
Background color in flakefinder overview report

### ISSUE THREAD

**Issue by fedepaol** [kubevirt/project-infra#174]
Once https://github.com/kubevirt/project-infra/pull/168 is merged, the results of the reports will be available in a view that looks like 
![flakefinder report](https://user-images.githubusercontent.com/809335/63693834-5b1ce700-c815-11e9-9646-3d60e3eb3e15.png )

It would be nice to have those cells with a background color that represent the overall score of that report (i.e. less flaky tests -> greener).
